sdk/ml/azure-ai-ml/azure/ai/ml/constants/_common.py

```diff
@@ -572,8 +572,6 @@ class ModelType:
 
 
 class RollingRate:
-    YEAR = "year"
```

Comment:
RollingRate was used exclusively by MDC and has been in preview mode from the start?
Else we should just leave the values here for backwards compat

Reply:
Rolling rate was exclusively for mdc